src/sage/algebras/commutative_dga.py

```diff
@@ -943,9 +949,9 @@ def __classcall__(cls, base, names=None, degrees=None, R=None, I=None, category=
 
         TESTS::
 
-            sage: A1 = GradedCommutativeAlgebra(GF(2), 'x,y', (3, 6))
-            sage: A2 = GradedCommutativeAlgebra(GF(2), ['x', 'y'], [3, 6])
-            sage: A1 is A2
+            sage: A1 = GradedCommutativeAlgebra(GF(2), 'x,y', (3, 6))                   # needs sage.rings.finite_rings
+            sage: A2 = GradedCommutativeAlgebra(GF(2), ['x', 'y'], [3, 6])              # needs sage.rings.finite_rings
+            sage: A1 is A2                                                              # needs sage.rings.finite_rings
             True
 
```

Comment:
GF(2) is small?

Reply:
Thanks. Done throughout the file in bd006df6786